javascript/central-login-oidc/src/main.js

```diff
@@ -11,9 +11,9 @@ import * as forgerock from '@forgerock/javascript-sdk';
  */
 
 const config = await forgerock.Config.setAsync({
-  clientId: process.env.WEB_OAUTH_CLIENT, // e.g. 'ForgeRockSDKClient'
+  clientId: process.env.WEB_OAUTH_CLIENT, // e.g. 'ForgeRockSDKClient' or PingOne Services Client GUID 
   redirectUri: `${window.location.origin}`, // Redirect back to your app, e.g. 'https://sdkapp.example.com:8443'
```

Comment:
Suggest we match the "e.g." example to where this sample actually runs, which I think is `http://localhost:8443` looking at the Webpack config? (not sure if it supports HTTPS or not?)